tools/mo/unit_tests/mo/utils/ir_reader/layer_to_class_test.py

```diff
@@ -6,85 +6,66 @@
+            **regular_op_with_shaped_data('conv', shape, {'type': 'Convolution'}),
+            **valued_const_with_data('weights', weights_const, weights_shape, {'type': 'Const'}),
+            **regular_op_with_shaped_data('reshape', reshape_shape, {'type': 'Reshape'}),
+            **shaped_const_with_data('reshape_const', len(reshape_shape) if reshape_shape is not None else None, {'type': 'Const'}),
+            **regular_op_with_shaped_data('add', shape, {'type': 'Add'}),
+            **shaped_const_with_data('add_const', [1, 32, 1, 1], {'type': 'Const'}),
+            **result("result")
         }
 
-        edges = [('input', 'input_data'),
-                 ('input_data', 'group_conv'),
-                 ('weights', 'weights_data'),
-                 ('group_conv', 'group_conv_data'),
-                 ('group_conv_data', 'add'),
-                 ('add_const', 'add_const_data'),
-                 ('add_const_data', 'add'),
-                 ('add', 'add_data'),
-                 ('add_data', 'result'),
+        edges = [*connect('input:0', 'group_conv:0'),
+                 *connect('group_conv:0', 'add:0'),
+                 *connect('add_const:0', 'add:1'),
+                 *connect('add:0', 'result:0'),
                  ]
 
         if reshape_shape is not None:
 
-            edges += [('weights_data', 'reshape'),
-                      ('reshape_const', 'reshape_const_data'),
-                      ('reshape_const_data', 'reshape'),
-                      ('reshape', 'reshape_data'),
-                      ('reshape_data', 'group_conv')]
+            edges += [*connect('weights:0', 'reshape:0'),
+                      *connect('reshape_const:0', 'reshape:1'),
+                      *connect('reshape:0', 'group_conv:1')]
         else:
-            edges.append(('weights_data', 'group_conv'))
+            edges += [*connect('weights:0', 'group_conv:1')]
 
-        graph = build_graph(nodes_attributes, edges, nodes_with_edges_only=True)
+        graph = build_graph(nodes_attributes, edges)
+        reshape_node = None
+        if reshape_shape is None:
+            reshape_node = Node(graph, 'reshape')
 
-        graph_ref = build_graph(nodes_attributes,
-                                [('input', 'input_data'),
-                                 ('input_data', 'conv'),
-                                 ('weights', 'weights_data'),
-                                 ('weights_data', 'conv'),
-                                 ('conv', 'conv_data'),
-                                 ('conv_data', 'add'),
-                                 ('add_const', 'add_const_data'),
-                                 ('add_const_data', 'add'),
-                                 ('add', 'add_data'),
-                                 ('add_data', 'result'),
-                                 ], nodes_with_edges_only=True)
+        for op in graph.get_op_nodes(type='GroupConvolution'):
+            groupconv_to_conv(op)
```

Comment:
How this is different from lines 69-70?

Reply:
This pass is not needed. Removed it, thanks.